deps/tools/common.mk

```diff
@@ -104,8 +104,9 @@ endif
 
 
 ## PATHS ##
+
 # sort is used to remove potential duplicates
-DIRS := $(sort $(build_bindir) $(build_depsbindir) $(build_libdir) $(build_includedir) $(build_sysconfdir) $(build_datarootdir) $(build_staging) $(build_prefix)/manifest)
+DIRS := $(sort $(build_bindir) $(build_depsbindir) $(build_libdir) $(build_includedir) $(build_sysconfdir) $(build_datarootdir) $(build_staging) $(build_prefix)$(MAYBE_HOST)/manifest)
```

Comment:
Doesn't `$(build_prefix)` already have `$(MAYBE_HOST)` embedded within it?

Reply:
Good catch.